### PR TITLE
[RELAY][OP] Add relay minimum op

### PR DESCRIPTION
**Pull request by mutinifni**
added relay op for elementwise minimum.

### CODE REVIEW

File: tests/python/relay/test_op_level4.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def test_cmp_type():
 
 
 def test_binary_broadcast():
-    for op in [relay.right_shift]:
+    for op in [relay.right_shift, relay.minimum]:
```
Comment (tmoreau89) [inline tests/python/relay/test_op_level4.py]: add this on a new line to make merging easier

File: docs/langref/relay_op.rst [hunk 2]
```diff
@@ -78,3 +78,4 @@ Level 4 Definitions
 .. autofunction:: tvm.relay.greater_equal
 .. autofunction:: tvm.relay.less
 .. autofunction:: tvm.relay.less_equal
+.. autofunction:: tvm.relay.minimum
```
Comment (tmoreau89) [inline docs/langref/relay_op.rst]: add under  **Level 4: Broadcast and Reductions**

**Comment by tqchen** [comment]
please rebase against the master to resolve conflicts

**Comment by tqchen** [comment]
@Mutinifni please rebase and resolve conflict with the master

**Pull request merged**